package.json

```diff
@@ -93,8 +93,8 @@
     "@storybook/addon-links": "4.0.0-alpha.20",
     "@storybook/addon-options": "4.0.0-alpha.20",
     "@storybook/react": "4.0.0-alpha.20",
-    "@svgr/rollup": "2.3.0",
-    "@svgr/webpack": "2.3.0",
+    "@svgr/rollup": "2.4.1",
+    "@svgr/webpack": "2.4.1",
```

Comment:
What do we need the webpack version for again?

Reply:
It's for storybook. When we run `yarn start`, we actually build the app with webpack 😢

Reply:
Ah snap. My bad. Cheers!